content/en/synthetics/browser_tests/actions.md

```diff
@@ -13,106 +15,190 @@ further_reading:
+
+To save the assertion, click **Apply**. For more information, see [Advanced Options][1].
+
+
+[1]: /synthetics/browser_tests/advanced_options#user-specified-locator
+{{% /tab %}}
+{{% tab "Test Active Page Content" %}}
 
 #### Test that some text is not present on the active page
 
-Asserts that some specific text is **NOT** present on the current page.
+Create this assertion step to have your browser test confirm that the text you specified in the `Value` field is **not** present on the current page being recorded.
+
+To save the assertion, click **Apply**.
+
+#### Test that some text is present on the active page
+
+Create this assertion step to have your browser test confirm that the text you specified in the `Value` field is present on the current page being recorded.
+
+To save the assertion, click **Apply**.
 
 #### Test the content of the URL of the active page
 
-Takes the URL of the last page that was interacted with, then asserts whether a specific value (such as `string`, `number`, or `regex`) is present within it.
+Create this assertion step to have your browser test take the URL of the last page that a user interacted with and verify whether a value you specified is present in the URL.
 
-#### Test that an element is present on the active page
+Enter a name for the step. You can specify that the content `should equal`, `should not equal`, `should contain`, `should not contain`, `should start with`, `should not start with`, `should match regex`, `should be empty`, `should not be empty`, `should be greater than`, `should be greater than or equal`, `should be less than`, or `should be less than or equal` a given value.
 
-Asserts that an element (such as a specific `span`, `div`, `h`, or `a`) is present on the current page.
+To save the assertion, click **Apply**. You can test for a value in the URL such as `string`, `number`, or `regex`.
+
+{{% /tab %}}
+
+{{% tab "Special Assertions" %}}
 
 #### Test that an email was received
 
-Asserts that an email was sent and whether specific values (such as `string`, `number`, or `regex`) are present within the email subject or body. This assertion leverages [email variables][4], you consequently first need to create an email variable to be able to use a `Test that an email was received` assertion.
+Web application journeys often involve emails being triggered and sent to users' mailboxes, such as an email verification after account creation, email sent to reset forgotten passwords, email sent to notify order confirmation, or email confirmation after contact form submission.
 
-#### Test your UI with custom JavaScript
+Create this assertion step to have your browser test confirm that your application's email mechanisms are working and verify that the values you specified such as `string`, `number`, or `regex` are present in the email subject or body. 
```

Comment:
I feel it is too much in this doc page.
let's keep a concise summary of the feature and have a link to a dedicated guide. 
it will also make this more discoverable as a standalone page will exist